src/components/dropdown/dropdown.js

```diff
@@ -322,6 +325,7 @@ class Dropdown extends React.Component {
    * @return {Void}
    */
   handleFocus = () => {
+    this.calculatePosition();
```

Comment:
maybe as part of the `else` block? wouldn't need to calculate it if focus is blocked?